cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -124,14 +124,18 @@ public void start(Promise<Void> start) {
             if (config.featureGates().kafkaNodePoolsEnabled())  {
                 // When node pools are enabled, we create the NodePool watch
                 startFutures.add(kafkaAssemblyOperator.createNodePoolWatch(namespace).compose(w -> {
-                    LOGGER.info("Opened KafkaNodePool watch for {} operator", kafkaAssemblyOperator.kind());
+                    LOGGER.info("Opened watch for {} operator", KafkaNodePool.RESOURCE_KIND);
                     watchByKind.put(KafkaNodePool.RESOURCE_KIND, w);
                     return Future.succeededFuture();
                 }));
             }
 
-            startFutures.add(AbstractConnectOperator.createConnectorWatch(kafkaConnectAssemblyOperator, namespace, config.getCustomResourceSelector()));
-            startFutures.add(kafkaRebalanceAssemblyOperator.createRebalanceWatch(namespace));
```

Comment:
not clear why adding the creation of the rebalance watcher is gone here :-/

Reply:
The watch is now created in the same way as for the other Assembly operators. so it basically moved to the loop on the line 113-122.